spec/libraries/azure_engine_spec.rb

```diff
@@ -28,12 +31,14 @@ def delete_message(queue_name, message_object_id, message_object_pop_receipt)
   module Entity
     module Queue
       class DummyMessage
-        def id
-          1
-        end
-
-        def pop_receipt
-          ''
+        attr_accessor :id
+        attr_accessor :message_text
```

Comment:
__[reek]__
* [Attribute] is a writable attribute [:ref](https://github.com/troessner/reek/blob/master/docs/Attribute.md)

<!-- Comment by [SideCI](https://sideci.com) -->